hydrus/parser/openapi_parser.py

```diff
@@ -22,6 +22,19 @@ def try_catch_replacement(block: Any, get_this: str, default: Any) -> str:
         return default
 
 
+def get_class_name(class_location):
+    return class_location[len(class_location)-1]
+
+
+def get_data_at_location(class_location, doc):
```

Comment:
Use mypy format and add proper DocStrings

Reply:
@xadahiya fixed , other mypy errors have been fixed in #210